cwms-data-api/src/main/java/cwms/cda/formatters/Formats.java

```diff
@@ -292,6 +294,7 @@ public static ContentType parseQueryParam(String queryParam, Class<? extends Cwm
         return retVal;
     }
 
+    @Deprecated
```

Comment:
Can you add javadoc comment here with @deprecated suggestion or explanation

Reply:
Added a javadoc comment and the deprecated bit